src/test/java/org/opensearch/knn/index/FaissIT.java

```diff
@@ -280,4 +282,14 @@ public void testEndToEnd_fromModel() throws Exception {
             assertEquals(numDocs - i - 1, Integer.parseInt(results.get(i).getDocId()));
         }
     }
+
+    @Override
+    protected Settings restClientSettings() {
+        return noStrictDeprecationModeSettingsBuilder().build();
+    }
+
+    @Override
+    protected RestClient getClient() {
+        return adminClient();
```

Comment:
Not sure if we should use admin client for all requests.

Reply:
let me check, if there will be more case where normal client is enough I'll update the logic and use admin client only where needed